src/index.tsx

```diff
@@ -1,5 +1,5 @@
 import "../style/css/index.css";
-import { notebook_utils as nb_utils } from "./notebook_utils";
+import { notebook_utils as nb_utils, notebook_utils } from "./notebook_utils";
```

Comment:
Lets try and keep the imports organized. Putting all the jupyter imports together in one block, and all the imports for our code in another block.

Reply:
I should have checked that, I've been experimenting a lot when adding new functionality and the imports get automatically added. I'll be sure to clean it up in my next PR